qa/tasks/cephfs/test_volumes.py

```diff
@@ -4161,6 +4162,44 @@ def test_subvolume_rm_with_snapshots(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def _create_files_on_sv(self, sv_dir):
+        self.mount_a.remount(cephfs_name=self.volname)
+
+        self.mount_a.run_shell(f'sudo dd if=/dev/zero of={sv_dir}/file1 '
+                                'bs=1 count=0 seek=1G')
+        self.mount_a.run_shell(f'sudo cp {sv_dir}/file1 {sv_dir}/file2')
+        self.mount_a.run_shell(f'sudo cp {sv_dir}/file1 {sv_dir}/file3')
+
+    def _note_files_in_sv(self, sv_dir):
+        return self.mount_a.run_shell(f'sudo tree {sv_dir}').stdout.getvalue()
+
+    def assert_sv_is_present_in_trash(self, sv, sv_dir, sv_tree):
+        x = self.mount_a.run_shell('sudo ls volumes/_deleting').stdout.getvalue().strip()
+        trash_sv_dir = sv_dir.replace('_nogroup', f'_deleting/{x}')
+        trash_sv_dir = trash_sv_dir.replace(sv, '')
+        sv_tree_new = self.mount_a.run_shell(f'sudo tree {trash_sv_dir}').stdout.getvalue()
+        self.assertTrue(sv_tree_new, sv_tree)
+
+    def test_subvolume_rm_purge_trash(self):
+        '''
+        Test that when mgr config option mgr/volumes/purge_trash is set to
+        true, running "ceph fs subvolume rm" will move the subvolume to trash
+        but not delete the subvolume from trash.
+        '''
+        v, sv = self.volname, self._generate_random_subvolume_name()
+        self.run_ceph_cmd(f'fs subvolume create {v} {sv}')
+        self.run_ceph_cmd('config set mgr mgr/volumes/purge_trash false')
+        sv_dir = self.mount_a.run_shell(f'ls volumes/_nogroup/{sv}/').\
+            stdout.getvalue().strip()
+        sv_dir = f'volumes/_nogroup/{sv}/{sv_dir}'
+        self._create_files_on_sv(sv_dir)
+        sv_tree = self._note_files_in_sv(sv_dir)
+
+        self.run_ceph_cmd(f'fs subvolume rm {v} {sv}')
+        self.assert_sv_is_present_in_trash(sv, sv_dir, sv_tree)
+        time.sleep(60)
+        self.assert_sv_is_present_in_trash(sv, sv_dir, sv_tree)
+
```

Comment:
This test case needs to extended to validate, if `purge_trash` is set to `true`, the subvolumes shouldbe deleted.

Reply:
This is marked resolved without an explanation. Were the comments incorporated?

Reply:
These tests were deleted long ago since we changed the approach on this PR. Therefore, marking this thread as resolved.